<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MMI_Server_DemonClass</class>
 <widget class="QMainWindow" name="MMI_Server_DemonClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MMI_Server_Demon</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Server_Start_Bt">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开启</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>66</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设备1：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设备2：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设备3：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设备4：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设备5：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设备6：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设备7：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设备8：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设备9：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设备10：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>551</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>设备编号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>收到信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>发送消息</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>231</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="read_1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="read_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="read_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="read_4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="read_5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="read_6">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="read_7">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="read_8">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="read_9">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="read_10">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>181</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="send_1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="send_2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="send_3"/>
     </item>
     <item>
      <widget class="QLineEdit" name="send_4"/>
     </item>
     <item>
      <widget class="QLineEdit" name="send_5"/>
     </item>
     <item>
      <widget class="QLineEdit" name="send_6"/>
     </item>
     <item>
      <widget class="QLineEdit" name="send_7"/>
     </item>
     <item>
      <widget class="QLineEdit" name="send_8"/>
     </item>
     <item>
      <widget class="QLineEdit" name="send_9"/>
     </item>
     <item>
      <widget class="QLineEdit" name="send_10"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>77</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="Send_Bt_1">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Send_Bt_2">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Send_Bt_3">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Send_Bt_4">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Send_Bt_5">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Send_Bt_6">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Send_Bt_7">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Send_Bt_8">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Send_Bt_9">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Send_Bt_10">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mmi_server_demon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Server_Start_Bt</sender>
   <signal>clicked()</signal>
   <receiver>MMI_Server_DemonClass</receiver>
   <slot>My_Server_Start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Send_Bt_1</sender>
   <signal>clicked()</signal>
   <receiver>MMI_Server_DemonClass</receiver>
   <slot>Dev_Send1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Send_Bt_2</sender>
   <signal>clicked()</signal>
   <receiver>MMI_Server_DemonClass</receiver>
   <slot>Dev_Send2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>My_Server_Start()</slot>
  <slot>Dev_Send1()</slot>
  <slot>Dev_Send2()</slot>
 </slots>
</ui>
